code/espurna/sensor.cpp

```diff
@@ -1155,6 +1181,21 @@ String magnitudeName(unsigned char type) {
         case MAGNITUDE_CO2:
             result = F("CO2");
             break;
+        case MAGNITUDE_CO2E:
+            result = F("CO2e");
+            break;
+        case MAGNITUDE_BVOC:
```

Comment:
@mcspr I'm wondering if bVOC is too specific. Maybe we normalize to VOC and let the consumer interpret the results?

Reply:
If the above e suffix means 'estimate', maybe this one needs it too?
Perhaps, also as _ESTIMATE

Reply:
Yeah.. both values are "equivalent" values based on their proprietary algorithms. bVOC being breath-equivalent VOC compound analysis. 

So maybe drop CO2e and BVOC and replace by standard CO2 and VOC with note on wiki?

Reply:
> So maybe drop CO2e and BVOC and replace by standard CO2 and VOC with note on wiki?

Ok.